src/styleguide/components/s-api-mock-test.vue

```diff
@@ -154,32 +160,40 @@
 
     computed: {
       /**
-       * Returns list of endpoints  (filtered by search).
+       * Returns the mapped list of endpoints.
        */
-      filteredEndpoints(): object[] | null {
-        const { search, handlers } = this;
+      mappedEndpoints(): IEndpoint[] | null {
+        const { handlers } = this;
 
         if (!handlers) {
           return null;
```

Comment:
suggestion: 🤔 wouldn't it make more sense to always return an array? This would make it easier to handle in following mappers/filters.